README.md

```diff
@@ -70,13 +70,13 @@ function pprint(obj) {
 
 ### Analytics
 
-List available analytics:
+List all available analytics:
 
 ```js
 let analytics = api.listAnalytics();
 ```
 
-Get documentation for an analytic:
+Get documentation for an analytic. Asign certain `analyticId` as argument:
```

Comment:
Typo on `asign`. Should be `assign`. Better version:

"Get documentation for an analytic, where the `analyticId` variable matches the analytic's ID".

Reply:
ok